eng/pipelines/templates/stages/archetype-cpp-release.yml

```
@@ -69,6 +69,67 @@ stages:
                           # we override the regular script path because we have cloned the build tools repo as a separate artifact.
                           ScriptPath: '$(Build.SourcesDirectory)/eng/common/scripts/copy-docs-to-blobstorage.ps1'
 
+          - ${{ if ne(artifact.options.skipPublishVcpkg, 'true') }}:
+            - deployment: PublishVcpkg
+              displayName: Publish to vcpkg
+              condition: ne(variables['Skip.PublishVcpkg'], 'true')
+              environment: github
+              # This step requires the tag step
+              dependsOn: TagRepository
+
+              strategy:
+                runOnce:
+                  deploy:
+                    steps:
+                      - checkout: self
+
+                      - task: Powershell@2
+                        inputs:
+                          filePath: eng/scripts/Initialize-VcpkgRelease.ps1
+                          arguments: >-
+                            -Username azure-sdk
+                            -Pass $(azuresdk-github-pat)
+                            -GitHubRepo Azure/azure-sdk-for-cpp
```

```suggestion
                            -GitHubRepo "$(Build.Repository.Name)"
```

See https://github.com/Azure/azure-sdk-tools/pull/1000 for reasons.